firefox-ios/Client/Coordinators/Router/RouteBuilder.swift

```diff
@@ -202,7 +196,7 @@ final class RouteBuilder {
             TelemetryWrapper.recordEvent(
                 category: .action,
                 method: .open,
-                object: isPrivate ?.mediumQuickActionPrivateSearch:.mediumQuickActionSearch
```

Comment:
should we also remove any mention of `mediumQuickActionPrivateSearch` since it's not longer being used?